src/Microsoft.TemplateEngine.TemplateLocalizer.Core/Microsoft.TemplateEngine.TemplateLocalizer.Core.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NETStandardTargetFramework)</TargetFramework>
+    <Description>The core API for Template Localizer tool.</Description>
+    <IsPackable>true</IsPackable>
+    <Nullable>enable</Nullable>
+    <LangVersion>9.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0-preview.3.21164.3" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0-preview.3.21164.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
please remove Newtonsoft.Json version here - it is managed globally in Version.props

Reply:
This triggered making https://github.com/dotnet/templating/issues/3104 :)

Reply:
This package reference isn't even needed. Thanks for catching this. Removed.